lib/suse-cloud-upgrade-4-to-5-post

```diff
@@ -187,6 +186,20 @@ if [ -f "/opt/dell/crowbar_framework/barclamps/pacemaker.yml" ]; then
   fi
 fi
 
+### Updating default session timeout in nova_dashboard
+proposals=$(crowbar nova_dashboard proposal list)
+if [ "$proposals" != "No current proposals" ]; then
+  echo_summary "Setting default Horizon session timeout."
```

Comment:
To keep current style of messages, you should end with `...`. Also, s/Setting/Updating/